Lib/tabnanny.py

```diff
@@ -105,14 +105,14 @@ def check(file):
         errprint("%r: Token Error: %s" % (file, msg))
         return
 
-    except SyntaxError as msg:
-        errprint("%r: Token Error: %s" % (file, msg))
-        return
-
     except IndentationError as msg:
         errprint("%r: Indentation Error: %s" % (file, msg))
         return
 
+    except SyntaxError as msg:
+        errprint("%r: Token Error: %s" % (file, msg))
```

Comment:
This was likely a copy paste error of line 105:

```suggestion
        errprint("%r: Syntax Error: %s" % (file, msg))
```